etc/ci/performance/test_all.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+case "$1" in
+  --servo)
+    engine=""
+    ;;
+  --gecko)
+    engine="--engine gecko"
+    ;;
+  *)
+    echo "Can't understand the argument, using the default servo engine"
+    echo "Usage: test_all.sh [--gecko]"
+    engine=""
+    ;;
+esac
+
+source venv/bin/activate
```

Comment:
When we run this on the builders we'll likely either already be inside the virtualenv from `mach` or have system-wide-installed libraries available, so please remove this line. Probably would be good to update the docs to mention this.